hybrasyl/Messaging/Admin.cs

```diff
@@ -20,7 +20,57 @@ public new static ChatCommandResult Run(User user, params string [] args)
         {
             if (Game.World.WorldData.TryGetValue<User>(args[0], out User target))
             {
-                return Success($"User {target.Name} Cookies\n\n{target.GetCookies()}\n\nSession Cookies\n\n{target.GetSessionCookies()}", MessageTypes.SLATE_WITH_SCROLLBAR);
+                string cookies = $"User {target.Name} Cookie List\n\n---Permanent Cookies---\n";
```

Comment:
There must be a better way to do this.

Reply:
re: Lines 23-73

Reply:
I'm happy to refactor this if you have a better suggestion, but since this is a diagnostic / administrative slash command that will be run rarely, I don't believe it is worth optimizing.